docs/introduction.md

```diff
@@ -37,17 +38,17 @@ We offer [Augmented Reality](/javascript/augmented-reality), a [3D Viewer](/java
   </p>
 </div>
 
-## What technologies does your SDKs support?
+## What technologies do you support?
```

Comment:
> `## What solutions does R2U offer?`

Talvez aqui então também tratar o `you` como `R2U`.

`## What technologies does R2U support?`